<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>181</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>181</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>181</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GoPro 4 Hero remote</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="folderButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>folder</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>56</y>
      <width>130</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>power off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="photoButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>112</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>take photo</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>108</y>
      <width>130</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>9</y>
      <width>290</width>
      <height>95</height>
     </rect>
    </property>
    <property name="openLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>156</y>
      <width>430</width>
      <height>10</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>430</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>120</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>object</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>124</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Object name:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
